src/thor/bidirectional_astar.cc

```diff
@@ -228,7 +234,8 @@ inline bool BidirectionalAStar::ExpandForwardInner(GraphReader& graphreader,
     return true; // This is an edge we _could_ have expanded, so return true
   }
 
-  const uint64_t localtime = 0; // Bidirectional is not yet time-aware
+  // TODO: actually use time_info
```

Comment:
we're not actually to the point yet where bidirectional makes time aware expansion choices. coming soon